src/AspNetCoreModuleV2/CommonLib/StringHelpers.h

```diff
@@ -27,6 +27,7 @@ std::wstring format(const std::wstring& format, Args ... args)
         {
             throw std::system_error(std::error_code(errno, std::system_category()));
         }
+        result.resize(size);
```

Comment:
Do you need the initial resize to `size + 1`?

Reply:
Yeah. The null terminator is still written to the string.

Reply:
Let's say we have the string `"Hello %s"` with `args="World"`. The size of the resulting string is 11 characters not including the null terminator. Next, we call `const size_t size = swprintf(nullptr, 0, format.c_str(), args ...);` which would return 11. We resize result to 11. Now, on the second call to snprintf, we need to pass in n+1 for the size of the new buffer because "The generated string has a length of at most n-1, leaving space for the additional terminating null character." This is fine to do as std::string will always have room for one more character.

So yes, you are right in that we can use size(), but we weren't passing the right thing into snprintf.